node-stringprep.cc

```diff
@@ -19,13 +19,13 @@ class StringPrep : public ObjectWrap {
 public:
   static void Initialize(Handle<Object> target)
   {
-    NanScope();
-    Local<FunctionTemplate> t = NanNew<FunctionTemplate>(New);
+    Nan::HandleScope scope;
+    Local<FunctionTemplate> t = Nan::New<FunctionTemplate>(New);
     NanAssignPersistent(stringprep_constructor, t);
```

Comment:
../node-stringprep.cc:24:50: error: ‘NanAssignPersistent’ was not declared in this scope

     NanAssignPersistent(stringprep_constructor, t);